ansible/roles/dev-desktop/files/team_login/src/main.rs

```diff
@@ -90,6 +90,14 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
                 .success(),
             "failed to set the default shell"
         );
+
+        // Set a user quota
+        assert!(
+            cmd("setquota", &["-u", &username, "50G", "51G", "0", "0", "/"])?
```

Comment:
Can we make this configurable from the playbook?

Reply:
We sure can! I went through two iterations, first setting the quota as an environment variable and then passing it as an argument to the CLI. I like the second option much more, since it makes the dependency explicit. The downside is that I needed/wanted to pull in `clap` to parse the arguments.